### PR TITLE
Apply `black` to `pyomo/common` directory `py` files

### PR DESCRIPTION
**Pull request by mrmundt**
## Summary/Motivation:
This is the first in a relatively long string of PRs to apply PEP8 standards via `black` to Pyomo. We are following the standard used by IDAES _except_ that we are adding the `-S` option: `Don't normalize string quotes or prefixes` and the `-C` option: `Don't use trailing commas as a reason to split lines.`

**NOTE**: All changes are NFC.

Full command used: `cd pyomo/common && black -S -C .`

:warning: Please squash this at merge. :warning:

## Changes proposed in this PR:
- Apply `black` to the `pyomo/common` directory `py` files
- Manually fix instances where two lines were merged and left `" "` or `' '`

### Legal Acknowledgement

By contributing to this software project, I have read the [contribution guide](https://pyomo.readthedocs.io/en/stable/contribution_guide.html) and agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the BSD license.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: pyomo/common/tests/test_timing.py [hunk 1]
```diff
@@ -45,38 +50,34 @@ def test_raw_construction_timer(self):
...
+            r"[0-9\.]+ elapsed seconds",
+        )
 
     def test_raw_transformation_timer(self):
         a = TransformationTimer(None)
         self.assertRegex(
             str(a),
-            r"TransformationTimer object for NoneType; "
-            r"[0-9\.]+ elapsed seconds")
+            r"TransformationTimer object for NoneType; " r"[0-9\.]+ elapsed seconds",
```
Comment (blnicho) [inline pyomo/common/tests/test_timing.py:68]: Quote reformatting

File: pyomo/common/tests/test_unittest.py [hunk 2]
```diff
@@ -80,39 +84,35 @@ def test_assertStructuredAlmostEqual_nan(self):
 
     def test_assertStructuredAlmostEqual_errorChecking(self):
         with self.assertRaisesRegex(
-                ValueError, "Cannot specify more than one of "
-                "{places, delta, abstol}"):
+            ValueError, "Cannot specify more than one of " "{places, delta, abstol}"
```
Comment (blnicho) [inline pyomo/common/tests/test_unittest.py:87]: Quote reformatting

File: pyomo/common/unittest.py [hunk 3]
```diff
@@ -132,11 +141,10 @@ def assertStructuredAlmostEqual(first, second,
 
     """
     if sum(1 for _ in (places, delta, abstol) if _ is not None) > 1:
-        raise ValueError("Cannot specify more than one of "
-                         "{places, delta, abstol}")
+        raise ValueError("Cannot specify more than one of " "{places, delta, abstol}")
```
Comment (blnicho) [inline pyomo/common/unittest.py:144]: Quote reformatting

File: pyomo/common/tests/test_config.py [hunk 4]
```diff
@@ -732,119 +763,148 @@ def test_immutable_config_value(self):
...
             formatting; like a bulleted list:
               - item 1
               - item 2
-            """)).declare_as_argument(group='Scenario definition')
-        sc.declare('merlion', ConfigValue(
-            default=False,
-            domain=bool,
-            description='Water quality model',
-            doc="""
+            """,
```
[the remark this answers is not in the document]
Comment (mrmundt) [reply pyomo/common/tests/test_config.py:795]: I agree, but how would you want it fixed?

**Comment by mrmundt** [comment]
:warning: Please squash this at merge. ⚠️

**Pull request merged**